src/main/java/teammates/ui/webapi/action/SearchCommentsAction.java

```diff
@@ -0,0 +1,40 @@
+package teammates.ui.webapi.action;
+
+import java.util.List;
+
+import teammates.common.datatransfer.FeedbackResponseCommentSearchResultBundle;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.exception.UnauthorizedAccessException;
+import teammates.common.util.Const;
+import teammates.ui.webapi.output.CommentSearchResultsData;
+
+/**
+ * Action: Showing the InstructorSearchPage for an instructor.
```

Comment:
The backend should have no knowledge about what the data is going to be used so we should not mention here. Instead, mention what the action does.